src/main/java/it/mulders/mcs/cli/Cli.java

```diff
@@ -65,9 +70,42 @@ public SearchCommand() {
 
         @Override
         public Integer call() {
-            searchCommandHandler.search(this.query, this.lastVersions);
+            System.out.printf("Searching for %s...%n", query);
+            searchCommandHandler.search(SearchQuery.search(this.query).withLimit(this.lastVersions).build());
             return 0;
         }
     }
 
+    @CommandLine.Command(
+            name = "class-search",
+            description = "Search artifacts in Maven Central",
+            usageHelpAutoWidth = true
+    )
+    public class ClassSearchCommand implements Callable<Integer> {
+
+        @CommandLine.Parameters(
+                arity = "1",
+                description = {
+                        "What to search for.",
+                        "This should be a class name. The result is the artifacts that contain the class.",
+                }
+        )
+        private String query;
+
+        @CommandLine.Option(
+                names = { "-f", "--full-name" },
+                negatable = true,
+                arity = "0",
+                description = "Show <count> last versions",
+                paramLabel = "<count>"
```

Comment:
I think it makes sense to have the `"-l" / "--last"` parameter for class-name search, too?